### DISCUSSION TITLE
[Projects beta] including issues/PRs that I don't own

### DISCUSSION THREAD

**Discussion by wsdewitt** [community/community#10377] [Projects and Issues]
I want to use a personal project board to organize my work across several organizations. How do I reference issues/PRs in repos I don't own, but have access to? When I type "#" I only see repos I own populating the list.

**Comment by micc83** [comment]
In my opinion a couple of things would be needed to handle this "Personal" scenario without changing the "per organization" nature of Projects Beta. 

**1. An easy way to switch between all the projects one is involved with**
This could be implemented with a "smart" select or dropdown at the top of the screen to navigate between projects of all the organizations, not only the one owning the current project.

**2. A dedicated section to manage all the assigned tasks from all the organizations**
This could be a dedicated "Me" project where all the tasks assigned to one from multiple organizations are summarized. The only issue with this approach would be the assignment of statuses and custom fields, as they both are customizable per project and would largely diverge. For custom fields I might suggest to just hide em from the personal "project" (this also open the ability to have both global and personal custom fields, for example a "time spent" field that I want to handle on my personal "project" but not show to others). Not sure instead how to handle statuses as having different statuses inside the same view would be pretty annoying.
